### DISCUSSION TITLE
회의록 1일차

### DISCUSSION THREAD

**Discussion by comaserious** [semi-project-team/PSMC#4] [General]
Todo
- [x] 피그마 최종 수정 (의사/ 치료사 위주)
- [x] 로그인 페이지 완성하기
- [x] nav 완성하기

**Comment by comaserious** [comment]
브랜치 이름 룰 정하기
의견있으면 올려주세요

**Reply by wjdals83** [reply]
예시)
새로운 기능 개발 시 -> "feature/new-login-page"
에러 수정 시 -> "errorfix/new-login-page"
예외 처리 시 -> "addexception/new-login-page"
이미지 파일 업로드 시 -> "common/img"
css -> "css/new-login-page"

**Reply by comaserious** [reply]
하고있는일/페이지 이름

**Comment by wjdals83** [comment]
공통 색상
lightblue
